core/dbt/config/runtime.py

```diff
@@ -48,6 +48,25 @@ def load_project(
     return project
 
 
+def load_profile(
```

Comment:
@MichelleArk  I imagine we would use something like this to generate the profile object, we maybe should refactor this code to pass in a loaded project dict directly instead of reading from disk, that way we can reuse that loaded project to also construct project object